src/lms-harmonizer/tests_integration_sql/procs/mssql/harmonize_sections/test_lmssections_canvas.py

```diff
@@ -3,119 +3,125 @@
 # The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
```

Comment:
These are effectively the same proc tests as before, but now the "act" is to run the full harmonizer.